ast/compile.go

```diff
@@ -117,6 +117,7 @@ type Compiler struct {
 	initialized             bool                          // indicates if init() has been called
 	debug                   debug.Debug                   // emits debug information produced during compilation
 	schemaSet               *SchemaSet                    // user-supplied schemas for input and data documents
+	schemaAnnotations       []*SchemaAnnotation           // user-supplied list of associations between value paths and schema definitions
```

Comment:
I'm wondering, do we really need to tie in annotations to apply the authz schema? We already have [functionality](https://www.openpolicyagent.org/docs/latest/policy-language/#passing-a-single-file-with--s) to globally map a single schema file to the input document without the use of annotations; could that be leveraged?

Reply:
Interesting. I'll look into that.

Reply:
Looks like we should be able to do this w/o annotations involved. Thanks for the tip @johanfylling!